broker/src/main/java/org/apache/rocketmq/broker/BrokerController.java

```diff
@@ -922,13 +922,9 @@ private void initialTransaction() {
     }
 
     private void initialAcl() {
-        if (!this.brokerConfig.isAclEnable()) {
-            LOG.info("The broker dose not enable acl");
-            return;
-        }
-
         List<AccessValidator> accessValidators = ServiceProvider.load(AccessValidator.class);
-        if (accessValidators.isEmpty()) {
+
+        if (accessValidators.isEmpty() && this.brokerConfig.isAclEnable()) {
```

Comment:
If the acl is turned off before starting the broker and the acl is turned on after starting the broker and the default validator is still empty, it is recommended that the initial add the default validator directly